scripts/unqueue.coffee

```diff
@@ -1,4 +1,5 @@
-require('babel-core/register')
+require('@babel/register')
```

Comment:
All of these individual `require('@babel/register')` lines can be removed since registration is happening in the root index file